File: app/src/main/java/com/o4x/musical/imageloader/glide/targets/AbsPaletteTargetListener.java
package com.o4x.musical.imageloader.glide.targets;

import android.graphics.Bitmap;

import androidx.annotation.Nullable;

import com.o4x.musical.imageloader.model.CoverData;

public class AbsPaletteTargetListener {

    public CoverData coverData;
    public boolean isSync = false;

    public void onResourceReady(@Nullable Bitmap resource) {}
}

File: app/src/main/java/com/o4x/musical/imageloader/glide/targets/palette/AbsPaletteTargetListener.kt
package com.o4x.musical.imageloader.glide.targets.palette

import android.content.Context
import android.graphics.Bitmap

open class AbsPaletteTargetListener(val context: Context) {

    @JvmField
    var isSync = false

    open fun onResourceReady(resource: Bitmap?) {}
}